api-spec/extensions/transaction/transaction.fragment.yaml

```diff
@@ -1,174 +1,211 @@
-              schema:
-                type: string
-    delete:
-      summary: delete an existing feature by Id
-      description: Use this method to delete an existing feature.
-      operationId: deleteFeature
-      tags:
-        - Extensions
-      parameters:
-        - $ref: '#/components/parameters/collectionId'
-        - $ref: '#/components/parameters/featureId'
-      responses:
-        '200':
-          description: Status of the update request.
-          content:
-            text/html:
-              schema:
-                type: string
-            application/json:
-              schema:
-                type: string
-        default:
-          description: An error occurred.
-          content:
-            application/json:
-              schema:
-                $ref: '#/components/schemas/exception'
-            text/html:
-              schema:
-                type: string
-components:
-  schemas:
-    partialItem:
-      type: object
-      properties:
-        id:
-          $ref: '#/components/schemas/itemId'
-        bbox:
-          $ref: '#/components/schemas/bbox'
-        geometry:
-          $ref: 'https://geojson.org/schema/Geometry.json'
-        type:
-          $ref: '#/components/schemas/itemType'
-        properties:
-          $ref: '#/components/schemas/partialItemProperties'
-        links:
-          $ref: '#/components/schemas/links'
-        assets:
-          $ref: '#/components/schemas/itemAssets'
-      example:
-        assets:
-          analytic:
-            title: 1-Band Analytic
-            href: http://cool-sat.com/catalog/collections/cs/items/CS3-20160503_132130_04/analytic-1.tif
-    partialItemProperties:
-      type: object
-      description: allows for partial collections of metadata fields
-      additionalProperties: true
-      properties:
-        datetime:
-          $ref: '#/components/schemas/time'
-tags:
-  - name: Extensions
-    description: >-
-      STAC-specific operations to add, remove, and edit items within WFS3
+openapi: 3.0.1
+info:
+  title: The SpatioTemporal Asset Catalog API + WFS3 + Transaction extension
+  version: 0.7.0
+paths:
+  '/collections/{collectionId}/items':
+    post:
+      summary: add a new feature to a collection
+      description: create a new feature in a specific collection
+      operationId: postFeature
+      tags:
+        - Extensions
+      parameters:
+        - $ref: '#/components/parameters/collectionId'
+      requestBody:
+        content:
+          application/json:
+            schema:
+                oneOf:
+                 - $ref: '#/components/schemas/item'
+                 - $ref: '#/components/schemas/itemCollection'
+      responses:
+        '201':
+          description: Status of the create request.
+          content:
+            application/geo+json:
+              schema:
+                type: string
+            text/html:
+              schema:
+                type: string
+        '400':
```

Comment:
For all of our API endpoints, I would like us to be explicit about what status codes are expected for which behaviors.